autoprotocol/instruction.py

```diff
@@ -1401,6 +1414,7 @@ def __init__(self, resource_id, dests, measurement_mode="volume"):
                 "measurement_mode": measurement_mode,
                 "to": dests,
             },
+            informatics={"informatics": informatics},
```

Comment:
Ideally.. I think we should allow `Informatics`-class here, rather than forcing someone to use a dict here

Reply:
updated :)